src/main/java/uk/gov/companieshouse/pscdataapi/service/CompanyPscService.java

```diff
@@ -169,4 +170,31 @@ public Individual getIndividualPsc(String companyNumber, String notificationId)
         }
     }
 
+    /** Get PSC record. */
+    /** and transform it into an individualBeneficialOwner PSC.*/
+    public IndividualBeneficialOwner getIndividualBeneficialOwnerPsc(
+            String companyNumber, String notificationId) {
+        try {
+            Optional<PscDocument> pscDocument =
+                    repository.findById(notificationId)
+                            .filter(document -> document.getData().getKind()
+                                    .equals("individual-beneficial-owner")
+                                    && document.getCompanyNumber().equals(companyNumber));
+            if (pscDocument.isEmpty()) {
+                throw new ResourceNotFoundException(HttpStatus.NOT_FOUND,
+                        "PSC document not found in Mongo with id " + notificationId);
```

Comment:
Lets make this log "Individual Beneficial Owner PSC document not found in Mongo with id " as we will also get here if the PSC with that id has a different type

Reply:
I didn't notice this when we did the Individual one, lets change that log message too in fact